src/main/resources/com/google/api/codegen/nodejs/method_sample.snip

```diff
@@ -80,68 +92,60 @@
         {@sampleMethodCallArgList(apiMethod)})
 @end
 
-@private methodCallSampleCodeForPagedResponse(apiMethod)
+@private sampleMethodCallArgListAndComma(apiMethod)
     @if sampleMethodCallArgList(apiMethod)
-        function callback(err, response, nextPageToken) {
-            if (err) {
-                console.error(err);
-                return;
-            }
-            // doThingsWith(response)
-            if (nextPageToken) {
-                // fetch the next page.
-                api.{@apiMethod.name}(\
-                    {@sampleMethodCallArgList(apiMethod)}, {pageToken: nextPageToken}, callback);
-            }
-        }
-        api.{@apiMethod.name}(\
-            {@sampleMethodCallArgList(apiMethod)}, {flattenPages: false}, callback);
+        {@sampleMethodCallArgList(apiMethod)}, {@""}
     @else
-        function callback(err, response, nextPageToken) {
-            if (err) {
-                console.error(err);
-                return;
-            }
-            // doThingsWith(response)
-            if (nextPageToken) {
-                // fetch the next page.
-                api.{@apiMethod.name}({pageToken: nextPageToken}, callback);
-            }
-        }
-        api.{@apiMethod.name}({flattenPages: false}, callback);
     @end
 @end
 
-@private methodCallSampleCodeWithReturnValue(apiMethod)
-    @if sampleMethodCallArgList(apiMethod)
-        api.{@apiMethod.name}(\
-            {@sampleMethodCallArgList(apiMethod)}).then(function(response) {
-            // doThingsWith(response)
-        }).catch(function(err) {
-            console.error(err);
-        })
-    @else
-        api.{@apiMethod.name}().then(function(response) {
-            // doThingsWith(response)
-        }).catch(function(err) {
+@private methodCallSampleCodeForPagedResponse(apiMethod)
+    var options = {autoPaginate: false};
+    function callback(responses) {
+        // The actual resources in a response.
+        var resources = responses[0];
+        // The next request if the response shows there's more responses.
+        var nextRequest = responses[1];
+        // The actual response object, in case want to refer.
+        // var rawResponse = responses[2];
+        for (var i = 0; i < resources.length; ++i) {
+            // doThingsWith(resources[i]);
+        }
+        if (nextRequest) {
+            return api.{@apiMethod.name}(nextRequest, options).then(callback);
```

Comment:
I liked the comment `// Fetch the next page.` since it clearly highlights how to use pagination. It might be good to consider adding it back.